client/my-sites/marketing/connections/service-description.jsx

```diff
@@ -173,6 +179,24 @@ class SharingServiceDescription extends Component {
 				args: { service: this.props.service.label },
 				context: 'Sharing: Publicize',
 			} );
+		} else if ( 'refresh-failed' === this.props.status ) {
+			if ( this.props.expires ) {
+				description = this.props.translate(
+					'We are unable to refresh your %(service)s token. Please reconnect before %(expiryDate)s',
```

Comment:
I'm not sure of the best message here. People aren't necessarily aware that we have to refresh the tokens. Suggestions are welcome!

Reply:
What about something like this: `Your %(service)s connection has expired. This happens sometimes. Click the "Reconnect" button to fix it before %(expiryDate)s.` To make it clear that this isn't their fault, but they have to take action to fix it.

Reply:
Yes, maybe we just say that it's 'expired', as they're more likely to understand the type of problem. Talking about 'refreshing tokens' seems like it would confuse people.

We should probably have something in the support docs to link to, as the main reason for this problem is that their account is private. Perhaps we can tackle that with the email notifications.